packages/opds-web-client/src/components/Root.tsx

```diff
@@ -64,6 +71,7 @@ export interface RootProps extends StateProps {
   fetchPage?: (url: string) => Promise<CollectionData>;
   Header?: new() => __React.Component<HeaderProps, any>;
   Footer?: new() => __React.Component<FooterProps, any>;
+  EntryPointsTabs?: new() => __React.Component<EntryPointsTabsProps, any>;
```

Comment:
I would rather have this prop be something more generic, like "CollectionHeader". You could put anything in there, it just happens to be entry points for the circ manager.